src/openrct2-ui/windows/ServerList.cpp

```diff
@@ -76,21 +78,32 @@ void JoinServer(std::string address);
 class ServerListWindow final : public Window
 {
 private:
-    char _playerName[32 + 1] = {};
+    u8string _playerName = {};
     ServerList _serverList;
     std::future<std::tuple<std::vector<ServerListEntry>, StringId>> _fetchFuture;
     uint32_t _numPlayersOnline = 0;
     StringId _statusText = STR_SERVER_LIST_CONNECTING;
+    bool _nameTextBoxInFocus = false;
 
     bool _showNetworkVersionTooltip = false;
     std::string _version;
 
+    void CheckTextBoxFocus()
```

Comment:
This name is not very descriptive. "Check" is one of the words you should almost never use, because it's so ambiguous.

```suggestion
    void OnFocusLoss()
```

Reply:
Yea, it was pretty bad in hindsight, I did sit there thinking what to name it but yea didn't really work lol. A good reminder of ambiguity though.

Reply:
Generally it's best to pick a name that describes what a function does, not how it is used. That makes the intent much clearer at first glance.